sway-lib-std/src/i128.sw

```diff
@@ -0,0 +1,145 @@
+library i128;
+
+use core::num::*;
+use ::assert::assert;
+use ::u128::U128;
+
+/// The 128-bit signed integer type.
+/// Represented as an underlying U128 value.
+pub struct i128 {
```

Comment:
One philosophical thing: should these instead be `I8`, `I16`, `I32`, `I64`, and `I128` (i.e. uppercase `I`)? They're structs after all, not primitives.

Reply:
Good point. Applied it.